src/mongodb/mongo_setup.py

```diff
@@ -1,11 +1,26 @@
-from mongoengine import connect, disconnect
+from motor.motor_asyncio import AsyncIOMotorClient
+from beanie import init_beanie
+from src.mongodb.mongodb_models.library_item_model import LibraryItemDocument
+from src.mongodb.mongodb_models.patron_model import PatronDocument
+from src.mongodb.mongodb_models.bills_model import BillDocument
+from src.env_utils import MongoDBSettings
 
+client: AsyncIOMotorClient
 
-# Connect to MongoDB function
-def connect_to_mongodb(settings):
-    connect(db=settings.mongo_db_name, host=settings.mongo_client)
 
+async def connect_to_mongodb():
+    global client
```

Comment:
why?

Reply:
i dont succeed to connect to mongodb without the global client using async.